src/main/java/com/controller/RegistrationController.java

```diff
@@ -19,6 +19,7 @@
 
 	@RequestMapping(value = "/registerPage", method = RequestMethod.GET)
 	public String registerPage(@ModelAttribute("register") RegistrationBean registrationBean, BindingResult result) {
+		System.out.println("API reached");
```

Comment:
you can remove the print statement and use logger instead.

Reply:
> @anand2398 you can remove the print statement or use logger instead.

Addressed this comment.